libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -133,7 +220,13 @@ class AP_Airspeed
     bool healthy(uint8_t i) const {
         return state[i].healthy && (fabsf(param[i].offset) > 0 || state[i].use_zero_offset) && enabled(i);
     }
-    bool healthy(void) const { return healthy(primary); }
+    bool healthy(void) const {
+        if (blend_enable == BLEND_FULL) {
+            return blend_state.active;
```

Comment:
This stops the EKF from updating the wind estimate (where it is already applying it's own innovation filters), which means if you ever false positive into this because of not having a wind estimate (say takeoff into 10 m/s wind) you will never get a wind estimate and blending will forever be disabled (as far as I can see)

Reply:
That can be argued for the numbers, but the point is correct.
I am open for options to overcome this.